### ISSUE TITLE
[Bug] Talkback reads ' percent s' in strings with placeholders replaced by inline icon

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/fenix#4174]
## Steps to reproduce

Have a new install
Enable Talkback on your device
Navigate through onboarding cards
Select ‘Browse privately’ Card

### Expected behavior

A correct content description should be read with an appropriate description of the inline icon.

### Actual behavior

The text with ‘percent s’ is being read.

### Device information

* Android device: Huawei P10 (Android 9 - Talkback version 7.3.0)  - should reproduce independently of the device

* Fenix version: 1.0.1929

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest Nightly build 190806 with Sony Xperia Z5 Premium (Android 7.1.1), and LG G7 Fit (Android 8.1).
